tripy/nvtripy/backend/mlir/utils.py

```diff
@@ -124,8 +124,9 @@ def make_mlir_tensor(
 OUTPUT_SEPARATOR = ";;<out>;;"
 
 
-def make_tensor_location(input_names: List[str], output_names: List[str]) -> ir.Location:
-    return ir.Location.name(f"{','.join(input_names)}{OUTPUT_SEPARATOR}{','.join(output_names)}")
+def make_tensor_location(input_names: List[str], output_names: List[str], metadata: "ir.StringAttr") -> ir.Location:
```

Comment:
nit: can we make `metadata` a `str` and create the `ir.StringAttr` in this method?